educator/src/Dscp/Educator/Web/Educator/Types.hs

```diff
@@ -30,7 +30,8 @@ import Fmt (build, (+|), (|+), listF)
 import Dscp.Core
 import Dscp.Crypto
 import Dscp.Educator.Web.Types
-import Dscp.Util.Servant (ForResponseLog (..))
+import Dscp.Util.Servant (ForResponseLog (..),
+                          buildShortResponseList, buildLongResponseList)
```

Comment:
They should be at same line I guess.
We generally use `stylish-haskell` to automatically format imports on save.